config/locales/activerecord.yml

```diff
@@ -45,7 +45,7 @@ en:
   job_summary_errors: &job_summary_errors
     job_summary:
       blank: Enter a job summary
-    about_school:
+    about_organisation:
       blank: Enter a description of the school
```

Comment:
Probably we need to change this line too with school/trust

Reply:
Sure. So you want two separate translations (is that the correct term?)  - about_school and about_organisation?

Reply:
Will be picked up by https://dfedigital.atlassian.net/browse/TEVA-1117